### PR TITLE
Adding my releasing experience to RELEASING.md

### PR DESCRIPTION
**Pull request by clebertsuconic**

### CODE REVIEW

File: RELEASING.md [hunk 1]
```diff
@@ -36,6 +36,16 @@ The file will be located under ./artemis-pom/RELEASE/
...
 
 
+## Checking out a new empty git repository
+
+Before starting make sure you clone a brand new empty the git repository as follows as the release plugin will use the upstream for pushing the tags:
+
+```sh
+git clone git://github.com/apache/activemq-artemis.git
+cd activemq-artemis
+git remote add upstream https://git-wip-us.apache.org/repos/asf/activemq-artemis.git
```
Comment (johnament) [inline RELEASING.md]: Maybe set `pushChanges` to false? http://maven.apache.org/maven-release/maven-release-plugin/prepare-mojo.html#pushChanges

File: RELEASING.md [hunk 2]
```diff
@@ -45,6 +55,26 @@ mvn clean release:prepare -DautoVersionSubmodules=true -Prelease
...
+
+### Resuming release upload
+
+If something happened during the release upload you may need to eventually redo the upload.
+
+There is a release.properties file that is generated during the release. In case you want to upload a previously tagged release, add this file as follows:
+
+- release.properties
+```
+scm.url=scm:git:https://github.com/apache/activemq-artemis.git
```
Comment (johnament) [inline RELEASING.md]: Thats a read only mirror... how can we push to that?

**Comment by clebertsuconic** [comment]
github is not letting me answer online now.. or even Tag. 

So, for the release.properties... after the release is prepared... release:perform will checkout the TAG and do from scratch. This is supposed to be read-only. We could specify the apache repo, but not really needed.. this is just if something gets screwed during the upload.


pushChanges to false? We haven't used that... but it could be a nice trick. how it works? it just sets the committs locally? and you have to push yourself?

**Comment by clebertsuconic** [comment]
github is not letting me answer online now.. or even Tag. 

So, for the release.properties... after the release is prepared... release:perform will checkout the TAG and do from scratch. This is supposed to be read-only. We could specify the apache repo, but not really needed.. this is just if something gets screwed during the upload.


pushChanges to false? I haven't used that... but it could be a nice trick. how it works? it just sets the committs locally? and you have to push yourself?

**Comment by clebertsuconic** [comment]
github is not letting me answer online now.. or even Tag. 

So, for the release.properties... after the release is prepared... release:perform will checkout the TAG and do from scratch. This is supposed to be read-only. We could specify the apache repo, but not really needed.. this is just if something gets screwed during the upload.


pushChanges to false? I haven't used that... but it could be a nice trick. how it works? it just sets the committs locally? and you have to push yourself?

File: RELEASING.md (hunk 2 above)
Comment (clebertsuconic) [inline RELEASING.md]: So, for the release.properties... after the release is prepared... release:perform will checkout the TAG and do from scratch. This is supposed to be read-only. We could specify the apache repo, but not really needed.. this is just if something gets screwed during the upload.

File: RELEASING.md (hunk 1 above)
Comment (clebertsuconic) [inline RELEASING.md]: @johnament  I have never used that... it seems a nice trick.

It just sets the commits locally?

Comment (johnament) [inline RELEASING.md]: Yep, then you can push wherever you want.  Usually we only push tags after the release is fully executed.  So its ok if its sitting out on your mirror.

File: RELEASING.md (hunk 2 above)
Comment (johnament) [inline RELEASING.md]: Ah.  I misread what you meant by release upload... you're talking about the upload to nexus.

Comment (clebertsuconic) [inline RELEASING.md]: I will ammend with the word nexus to make sure it's clearer.

File: RELEASING.md (hunk 1 above)
Comment (clebertsuconic) [inline RELEASING.md]: @johnament how do you use this? from what I looked I would need to change the pom for that. Can I use -D variables somehow?

Comment (clebertsuconic) [inline RELEASING.md]: (added a note for that).


Should be ready to merge

**Pull request closed without merging**